doc/api/deprecations.md

```diff
@@ -2514,6 +2514,19 @@ Type: Documentation-only
 Prefer [`response.socket`][] over [`response.connection`] and
 [`request.socket`][] over [`request.connection`].
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: ClientRequest.abort() is destroy
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/29192
+    description: Documentation-only deprecation.
+-->
+
+[`ClientRequest.destroy()`][] should be the same as
+[`ClientRequest.abort()`][]. Make `ClientRequest` more streamlike by deprecating
+`abort()`.
+
```

Comment:
This reads more like a commit message than user documentation.

Reply:
how about now?

Reply:
Better thank you